.github/workflows/build.yaml

```diff
@@ -6,71 +6,82 @@ on:
 
 jobs:
     phpstan:
-        runs-on: ubuntu-20.04
+        runs-on: ubuntu-latest
```

Comment:
this should be 22.04 instead: we don't want our build to break because of an external change

Reply:
Sure, done!